include/dali/core/bitmask.h

```diff
@@ -126,14 +134,30 @@ class bitmask {
         *storage &= ~mask;
       return *this;
     }
+    bitref &operator|=(bool value) {
+      if (value)
+        *storage |= mask;
+      return *this;
+    }
+    bitref &operator&=(bool value) {
+      if (!value)
+        *storage &= ~mask;
+      return *this;
+    }
+    bitref &operator^=(bool value) {
+      if (value)
+        *storage ^= mask;
+      return *this;
+    }
```

Comment:
Shouldn't you add a test for it?

Reply:
Meh. I could... that's rather trivial, though.

Reply:
Done in index test.